src/core/types/workspace.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface WorkspaceAttribute {
```

Comment:
if this a core type would be able to have a fast follow to use this one within the [workspace plugin](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5075)?

Reply:
Gotcha! So it's here for the sake of breaking down the PRs. Thanks!